### PR TITLE
Code smell for loop braces

### PR DESCRIPTION
**Pull request by Rockdtben**
Fixes 7 Code Smells

### CODE REVIEW

File: Robust.Client/Console/Commands/Debug.cs [hunk 1]
```diff
@@ -312,10 +312,10 @@ public bool Execute(IDebugConsole console, params string[] args)
 
             var type = reflection.LooseGetType(args[1]);
             var getResourceMethod =
-                resourceCache.GetType().GetMethod("GetResource", new[] {typeof(string), typeof(bool)});
+                resourceCache.GetType().GetMethod("GetResource", new[] { typeof(string), typeof(bool) });
```
Comment (PJB3005) [inline Robust.Client/Console/Commands/Debug.cs:315]: Man we really need to configure IDE formatters to be consistent on this. I know my IDE auto format (Rider) will completely undo these changes.

Comment (Rockdtben) [reply Robust.Client/Console/Commands/Debug.cs:315]: I think there is a way to configure the style settings per solution in Visual Studio.

Is there a standard auto formatting you want for this project?

Comment (PJB3005) [reply Robust.Client/Console/Commands/Debug.cs:315]: I don't have too much of a preference on this kind of stuff, but it definitely should be consistent.

Rider can also pull formatting config from the solution (and also editorconfig). There are some general C# specific rules you can put into editorconfig which I believe VS should listen to aswell, but the one that would be relevant here is `csharp_space_within_single_line_array_initializer_braces` which only Rider/Resharper seem to care about.
